core/utils/docker.py

```diff
@@ -0,0 +1,32 @@
+from core.utils.process import Process
+from core.utils.run import run
+from core.utils.os_utils import OSUtils
+from core.log.log import Log
+from core.settings import Settings
+from core.enums.os_type import OSType
+
+
+class Docker(object):
+
+    @staticmethod
+    def start():
+        if Settings.HOST_OS == OSType.WINDOWS:
+            cmd = r'"C:\Program Files\Docker\Docker\Docker Desktop.exe"'
```

Comment:
May we do it this way:
- Have DOCKER_PATH in settings
- Read it from env variable and fallback to this value as default if variable is not set (on my machine path is different).

Reply:
Agree, will do it this way

Reply:
May be also ok to read env varialbe in this file, just to have the variable somewhere.